handlers/registration.js

```diff
@@ -1,77 +1,68 @@
 'use strict';
-
 const AWS = require('aws-sdk');
 const docClient = new AWS.DynamoDB.DocumentClient();
+const helpers = require('./helpers')
 
 module.exports.create = async (event, ctx, callback) => {
 
-  const timestamp = new Date().getTime();
+  // TODO: merge Jacques PR for checking required fields
   const data = JSON.parse(event.body);
-  const studentID = parseInt(data.studentID, 10);
-
-  const params = {
-      Item: {
-          studentID,
-          eventID: data.eventID,
-          status: data.status,
-          createdAt: timestamp,
-          updatedAt: timestamp
-      },
-      TableName: 'biztechRegistration' + process.env.ENVIRONMENT
-  };
-
-  await docClient.put(params).promise()
-
-  // Update Event count
-  let updateExpression = 'set ';
-
-  if (data.status == 'cancelled') {
-      updateExpression += 'registeredNum \= registeredNum - :incr,';
-  } else {
-      const num = data.status + 'Num';
-      updateExpression +=  num + ' \= ' + num + ' \+ :incr,';
-  }
-
-  let expressionAttributeValues = {':incr': 1};
-
-  // Update timestamp
-  updateExpression += "updatedAt = :updatedAt";
-  expressionAttributeValues[':updatedAt'] = timestamp;
-
-  // Log the update expression
-  console.log(updateExpression);
+  const id = parseInt(event.queryStringParameters.id, 10);
+  const eventID = data.eventID;
+  let registrationStatus = data.registrationStatus;
 
   const eventParams = {
-    Key: {
-      id: data.eventID
-    },
-    TableName: 'biztechEvents' + process.env.ENVIRONMENT,
-    ExpressionAttributeValues: expressionAttributeValues,
-    UpdateExpression: updateExpression,
-    ReturnValues:"UPDATED_NEW"
-  };
-
-  await docClient.update(eventParams).promise()
-  .then(result => {
-      const response = {
-        statusCode: 200,
-        headers: {
-          'Access-Control-Allow-Origin': '*',
-          'Access-Control-Allow-Credentials': true,
-        },
-        body: JSON.stringify('Update succeeded')
-      };
-      callback(null, response);
-  })
-  .catch(error => {
-    console.error(error);
-    const response = {
-      statusCode: 500,
-      body: error
+    Key: { id: eventID },
+    TableName: 'biztechEvents' + process.env.ENVIRONMENT
+  }
+  // Check if the event is full
```

Comment:
I don't think any of the tests covers this case, may be good to add one

Reply:
bet